packages/frontend-plugin-api/src/extensions/createPageExtension.tsx

```diff
@@ -79,7 +79,10 @@ export function createPageExtension<
       const ExtensionComponent = lazy(() =>
         options
           .loader({ config, inputs })
-          .then(element => ({ default: () => element })),
+          .then(element => ({ default: () => element }))
+          .catch(error => ({
+            default: () => <ExtensionError error={error} />,
```

Comment:
Hmm, not sure about this pattern tbh, feel like it's actually a bit nicer to have this bubble up to error boundaries instead?

Reply:
Agree, refactored to bubble the error up: https://github.com/backstage/backstage/pull/20849/commits/821caba297663f0c58dfbddc741edda7c503a91c

I'm still creating a `BootErrorPage` component extension, should we remove it?

Reply:
Boot error removed here https://github.com/backstage/backstage/pull/21652